project_4/index.html

```diff
@@ -0,0 +1,384 @@
+            </div>
+
+            <a href="#register" class="home-page-register">Register</a>
+
+        </header>
+
+        <!-- MISSION -->
+        <section id="mission" class="mission">
+            <div class="left-column">
+                <h2 data-aos="fade-right">What is Our Mission?</h2>
+                <p class="mission-text">The Sustainable By Design Conference aims to define sustainable design,
+                    understanding that
+                    sustainability pertains to the environment, social welfare, and economic welfare. We will reevaluate
+                    current design practices in the context of our ourselves, our communities, and the natural world. We
+                    hope to address pressing issues related to the way in which our world is currently heading and offer
+                    practical design solutions and goals to change our path towards a more sustainable one. This
+                    conference will also allow us to open up a dialogue and work together to find better solutions in
+                    making the design world, and the rest of the world, more sustainable. </p>
+            </div>
+            <div class="right-column">
+                <h2 data-aos="fade-left"> Who should come?</h2>
+                <p class="mission-text">Of course, anyone is welcome to come and learn sustainabile design if that if
+                    your passion. We
+                    believe every designer should be informed about sustainability, and the cost of our actions if we
+                    continue designing in the way we design now. Sustainanbility should be on every designer's mind
+                    before they begin a project whether you are a graphic designer, fashion designer, or a product
+                    designer; it's all
+                    relevant. It is also important to hear from a variety of practicers and perspectives from every
+                    field, because believe it or not, a lot of fields of design are interconnected; it is important
+                    to learn about their practices moving forward in the age of our ever changing climate and
+                    economy.</p>
+            </div>
+
+            <div class="quotes">
+                <h2 data-aos="fade-right">What Are People Saying?</h2>
+                <blockquote class="quote">"From top to bottom the speakers and topics were carefully curated. The
+                    execution was flawlessly organized. I made so many connections. Professional development has never
+                    been so fun."</blockquote>
+                <!-- https://revolveconference.com/ -->
+                <p> - Douglas Davis, Creative Strategist - </p>
+                <button type="button" class="button-reset button-next">Next</button>
+            </div>
+        </section>
+
+        <!-- SPEAKERS -->
+        <h2 class="section-header">Speakers</h2>
```

Comment:
Ideally, your heading element goes inside your `section` and is considered as a label/title for that section.